tests/test_filereader.py

```diff
@@ -14,7 +14,7 @@
 import sys
 import tempfile
 import unittest
-from warncheck import assertWarns
+from .warncheck import assertWarns
```

Comment:
As per my comment in the discussion, can we go to full absolute import for this and the other lines?  Are there any issues for pytest?